spinner-lib/src/main/AndroidManifest.xml

```diff
@@ -2,7 +2,7 @@
     package="com.duchuyctlk.widget">
 
     <application
-        android:allowBackup="false"
+        android:allowBackup="true"
```

Comment:
No. AllowBackup = false does not allow the use of the library with my application, which requires allowBackup = true.